src/Turbo/README.md

```diff
@@ -290,8 +289,8 @@ class ChatController extends AbstractController
             $form = $emptyForm;
         }
 
-        return $this->render('chat/index.html.twig', [
-            'form' => $form->createView(),
+        return $this->renderForm('chat/index.html.twig', [
```

Comment:
IMHO we should either only use `renderForm()` everywhere (as here) or show how to use `renderForm()` and the old way everywhere.

I'm ok with both approaches.